src/libraries/System.Private.CoreLib/src/System/DateTime.cs

```diff
@@ -332,6 +332,7 @@ public DateTime(int year, int month, int day, int hour, int minute, int second)
             else
             {
                 // if we have a leap second, then we adjust it to 59 so that DateTime will consider it the last in the specified minute.
+                // codeql[cs/leap-year/unsafe-date-construction-from-two-elements] - DateTime is constructed using the user specifed values, not a combination of different sources.
```

Comment:
```suggestion
                // codeql[cs/leap-year/unsafe-date-construction-from-two-elements] - DateTime is constructed using the user specified values, not a combination of different sources.  It would be intentional to throw if an invalid combination occurred.
```